package.json

```diff
@@ -37,7 +37,7 @@
     "babel-plugin-jsx-dom-expressions": "^0.38.5",
     "coveralls": "^3.1.1",
     "csstype": "^3.1.0",
-    "dom-expressions": "0.38.5",
+    "dom-expressions": "https://gitpkg.vercel.app/trusktr/dom-expressions/packages/dom-expressions?update-exports",
```

Comment:
This is a quick way to test a package from any git repo, including sub-folders.

See https://gitpkg.vercel.app

Reply:
All tests are passing. Anything I may not have thought of that needs testing?